src/exprestive.coffee

```diff
@@ -89,7 +89,7 @@ class Exprestive
 
   # Save a function to @reversePaths to get a url back from a route name
   registerReverseRoute: ({routeName, url}) ->
-    @reversePaths[camelCase routeName] = (id) ->
+    @reversePaths[routeName] = (id) ->
```

Comment:
For projects like pairforce that are using snake case. I think it would be nice if you could call the restful routes with either so you can match your own variable scheme.